discoveries/air-quality-so2-volcanoes.discoveries.mdx

```diff
@@ -0,0 +1,162 @@
+  </Chapter>
+  <Chapter
+    center={[-31.6486, 63.454738]}
+    zoom={3}
+    datasetId='so2'
+    layerId='OMSO2PCA-COG'
+    datetime='2021-01-01'
+  >
+    Data is primarily collected from three sources: the OMI ([Ozone Monitoring Instrument](https://aura.gsfc.nasa.gov/omi.html#:~:text=About%20OMI,data%20to%20derive%20tropospheric%20ozone.)) on the NASA [EOS Aura](https://aura.gsfc.nasa.gov/) satellite since 2004; the [Ozone Mapping and Profiler Suite](https://ozoneaq.gsfc.nasa.gov/omps/) (OMPS) on NASA-NOAA’s [NPP Suomi](https://www.nasa.gov/mission_pages/NPP/main/index.html) satellite, which launched in 2011; and the [Tropospheric Monitoring Instrument](http://www.tropomi.eu/) (TROPOMI) on the ESA Copernicus Sentinel-5 Precursor satellite since 2017. These satellites retrieve SO2 vertical column densities (VCDs) from measurements in the UV part of the spectrum. The retrieved SO2 VCDs correspond to 1 km thick sulfur dioxide plumes near the surface. _Shown on the map: Eruption of Fagradalsfjall in Iceland in 2021._
+  </Chapter>
+</ScrollytellingBlock>
+
+<Block>
+  <Figure>
+    <iframe
+      width="100%"
+      height="100%"
+      style={{ aspectRatio: "70/50" }}
+      src="https://maps.disasters.nasa.gov/arcgis/apps/webappviewer/index.html?id=514e11b43f9b493c832c6520f653e523"
+      title="Volcanoes Around the World"
+      frameborder="0"
+
+    />
+    <Caption>
+      Locations of SO2 emissions, including volcanoes, around the world that NASA monitors.
+    </Caption>
+  </Figure>
+</Block>
+
+<Block>
+  <Prose>
+    Volcanoes are not the only source of SO2; researchers closely monitor emissions from power plants, smelters (where metals are refined), and oil and gas industry-related sources. 
+
+    The current catalog of major emissions consists of 759 sources, including 477 power plants, 74 smelters, 102 oil and gas-related sources, and 106 volcanoes. This steeply increased from the first catalog, introduced in 2016, which logged 491 sources. This increase can be attributed to the addition of TROPOMI data, which made it possible to track smaller sources. Emissions remained relatively stable for the first 15 years of tracking before declining to 40% in 2019-2021. This transformation can largely be attributed to a massive decline in emissions from volcanic sources in the Pacific: Kilauea in Hawaii; and Aoba and Ambrym in Vanuatu. The data indicates a decline of approximately 50% overall in global SO2 emissions between 2005 and 2021. 
+  </Prose>
+  <Figure>
+    <Chart
+      dataPath={new URL('./so2-emissions.csv', import.meta.url).href}
+      dateFormat="%Y" 
+      idKey='Type' 
+      xKey='Year' 
+      yKey='Emissions' 
+      highlightStart = '2005'
```

Comment:
Is this highlight necessary? It looks like the entire timeline is selected. @kcarini

Reply:
no idea. i just copied formatting for charts from the doc. is `highlightStart` only required if your chart is for a subset of the dataset? should line 109 be removed entirely, is that what you mean? @hanbyul-here 

<img width="876" alt="Screen Shot 2023-01-23 at 10 07 39 AM" src="https://user-images.githubusercontent.com/9040572/214103689-0f16e22f-6bdb-4542-820a-38e31040198c.png">

Reply:
`highlight` related properties are only required if the chart has a reference/highlight area.  (area with sky blue background). It sounds like this chart doesn't need one, we can get rid of the lien from 109 to 111, 
![Screen Shot 2023-01-23 at 12 44 23 PM](https://user-images.githubusercontent.com/4583806/214111569-01ba7a6d-5277-4dd9-9dc7-006752565923.png)